website-custom-views/src/content/concepts/merchant-center-proxy-router.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Merchant Center Proxy Router
```

Comment:
I'm not sure if it makes sense to bring this file to the Custom Views as I think it's not relevant for their environments.

Happy to read your opinions.